Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerView.swift

```diff
@@ -48,31 +48,22 @@ open class _ChatMessageComposerView<ExtraData: ExtraDataTypes>: _View, ThemeProv
         .withoutAutoresizingMaskConstraints
     
     public private(set) lazy var sendButton = components
-        .messageComposer
         .sendButton.init()
         .withoutAutoresizingMaskConstraints
     
     public private(set) lazy var confirmButton: UIButton = UIButton()
         .withoutAutoresizingMaskConstraints
     
-    public private(set) lazy var attachmentButton: UIButton = components
-        .messageComposer
-        .composerButton.init()
+    public private(set) lazy var attachmentButton: UIButton = UIButton()
         .withoutAutoresizingMaskConstraints
     
-    public private(set) lazy var commandsButton: UIButton = components
-        .messageComposer
-        .composerButton.init()
+    public private(set) lazy var commandsButton: UIButton = UIButton()
         .withoutAutoresizingMaskConstraints
     
-    public private(set) lazy var shrinkInputButton: UIButton = components
-        .messageComposer
-        .composerButton.init()
+    public private(set) lazy var shrinkInputButton: UIButton = UIButton()
         .withoutAutoresizingMaskConstraints
     
-    public private(set) lazy var dismissButton: UIButton = components
-        .messageComposer
-        .composerButton.init()
+    public private(set) lazy var dismissButton: UIButton = UIButton()
```

Comment:
My idea is to refactor this and extract all these other buttons to their own button components as we have with `SendButton` and `ConfirmButton`, and probably have `Components.buttons` just like we have with `Components.images` and `Components.fonts`, WDYT?

Reply:
@nuno-vieira I don't we need `Components.buttons`. I kinda like the "keep-it-flat" rule :) It's simple and you don't have to think about it. It's also easier to see access the components via autocompletion like that.

Reply:
Okay, my only concern is that this will be a flat list of maybe 50+ components. The autocompletion will be hard to find what you need. So the idea that I was having was to replicate the structure of the design system. We have images, icons, fonts, buttons, inputs, etc... But lets make it flat for now, and later if we think is useful, we can change it 👍